### ISSUE TITLE
Version increment

### ISSUE THREAD

**Issue by srid** [tomsmalley/semantic-reflex#12]
I think the cabal version should be incremented when there is an API change, so I can specify a min_version in my app's cabal file. What do you think?

**Comment by tomsmalley** [comment]
I agree, it's probably time to do this properly now people are depending on the library
